### ISSUE TITLE
Updating the container causes it to fail to connect

### ISSUE THREAD

**Issue by Cruv** [binhex/arch-qbittorrentvpn#58]
Hello,

It seems when I update the container image I run into this issue where it will attempt to start, fail to make a connection, then restart. I always get it working eventually but I can't seem to pinpoint the cause. Below is the output from the log. I have tried opening the port 1198 on my firewall as well as mapping it to the container in Portainer. Any help is greatly appreciated.

Wed Oct 28 11:27:26 2020 [UNDEF] Inactivity timeout (--ping-restart), restarting
2020-10-28 11:27:26,576 DEBG 'start-script' stdout output:
Wed Oct 28 11:27:26 2020 SIGHUP[soft,ping-restart] received, process restarting
2020-10-28 11:27:26,578 DEBG 'start-script' stdout output:
Wed Oct 28 11:27:26 2020 WARNING: file 'credentials.conf' is group or others accessible
Wed Oct 28 11:27:26 2020 OpenVPN 2.4.9 [git:makepkg/9b0dafca6c50b8bb+] x86_64-pc-linux-gnu [SSL (OpenSSL)] [LZO] [LZ4] [EPOLL] [PKCS11] [MH/PKTINFO] [AEAD] built on Apr 20 2020
Wed Oct 28 11:27:26 2020 library versions: OpenSSL 1.1.1h  22 Sep 2020, LZO 2.10
2020-10-28 11:27:31,579 DEBG 'start-script' stdout output:
Wed Oct 28 11:27:31 2020 NOTE: the current --script-security setting may allow this configuration to call user-defined scripts
2020-10-28 11:27:31,579 DEBG 'start-script' stdout output:
Wed Oct 28 11:27:31 2020 CRL: loaded 1 CRLs from file [[INLINE]]
2020-10-28 11:27:31,579 DEBG 'start-script' stdout output:
Wed Oct 28 11:27:31 2020 TCP/UDP: Preserving recently used remote address: [AF_INET]x.x.x.x:1198
2020-10-28 11:27:31,579 DEBG 'start-script' stdout output:
Wed Oct 28 11:27:31 2020 UDP link local: (not bound)
Wed Oct 28 11:27:31 2020 UDP link remote: [AF_INET]x.x.x.x:1198
2020-10-28 11:28:31,827 DEBG 'start-script' stdout output:
Wed Oct 28 11:28:31 2020 [UNDEF] Inactivity timeout (--ping-restart), restarting
2020-10-28 11:28:31,840 DEBG 'start-script' stdout output:
Wed Oct 28 11:28:31 2020 SIGHUP[soft,ping-restart] received, process restarting
2020-10-28 11:28:31,840 DEBG 'start-script' stdout output:
Wed Oct 28 11:28:31 2020 WARNING: file 'credentials.conf' is group or others accessible
Wed Oct 28 11:28:31 2020 OpenVPN 2.4.9 [git:makepkg/9b0dafca6c50b8bb+] x86_64-pc-linux-gnu [SSL (OpenSSL)] [LZO] [LZ4] [EPOLL] [PKCS11] [MH/PKTINFO] [AEAD] built on Apr 20 2020
Wed Oct 28 11:28:31 2020 library versions: OpenSSL 1.1.1h  22 Sep 2020, LZO 2.10
2020-10-28 11:28:36,829 DEBG 'start-script' stdout output:
Wed Oct 28 11:28:36 2020 NOTE: the current --script-security setting may allow this configuration to call user-defined scripts
2020-10-28 11:28:36,843 DEBG 'start-script' stdout output:
Wed Oct 28 11:28:36 2020 CRL: loaded 1 CRLs from file [[INLINE]]
2020-10-28 11:28:36,843 DEBG 'start-script' stdout output:
Wed Oct 28 11:28:36 2020 TCP/UDP: Preserving recently used remote address: [AF_INET]x.x.x.x:1198
Wed Oct 28 11:28:36 2020 UDP link local: (not bound)
Wed Oct 28 11:28:36 2020 UDP link remote: [AF_INET]x.x.x.x:1198

**Comment by Tuumke** [comment]
+1 for me

**Comment by Tuumke** [comment]
Using nextgen openvpn file seems to fix the connection. Just says port forward is not enabled

**Comment by cameronwc** [comment]
Updating to the nextgen openvpn files fixed the problem and mine says port forwarding is enabled. Furthermore I had to extract the pia nextgen files and remove all of the .ovpn files I didn't want to use. I also had to rename the ovpn file to not have a space in it.

**Comment by Cruv** [comment]
I got it to work with the nextgen files as well. The only thing that tripped me up was that I am using PIA and they had a separate username and password from the one you log into their dashboard with that you used to have to use for the old L2TP config. After fumbling around and figuring out I had to change it to the same user/pass I use to log into PIA, I got it working. Leaving it up to the author Binhex to decide if putting a warning in the log is worth it or something to someone. Otherwise this can be closed.
